test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,89 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+describe('First Api Tests', () => {
+});
+
+it('Consume GET Service', async () => {
+    const response = await axios.get('https://httpbin.org/ip');
+  
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data).to.have.property('origin');
+    
```

Comment:
Avoid this empty lines at the end of functions or methods

Reply:
Done!